rfc-012-makefile-standards.md

```diff
@@ -0,0 +1,64 @@
+# Makefile standards
+
+## Summary
+
+A set of standards for Makefiles across all projects
+
+## Problem
+
+Having a standard set of commands that can be run across all projects for common tasks
+allows people to get projects up and running without having to spend time looking into
+how things are set up to know what specific commands to run.
+
+
+## Proposal
+
+All projects MUST have a Makefile.
```

Comment:
Does this include NPM and RubyGem libraries? Or just services?

Reply:
`make test` for gems seems like a valid thing. NPM has `npm test` but for consistently `make test` shouldn't hurt. I'd say use Make everywhere – it doesn't matter what you have under the hood... clang, gcc, g++, npm, bundle exec.